hack/ci/e2e.sh

```diff
@@ -103,12 +103,27 @@ create_cluster() {
 # necessary for conformance
 kind: Cluster
 apiVersion: kind.sigs.k8s.io/v1alpha3
+kubeadmConfigPatches:
+- |
+  apiVersion: kubeadm.k8s.io/v1beta1
+  kind: ClusterConfiguration
+  metadata:
+    name: config
+  apiServer:
+  extraArgs:
+    "audit-policy-file": "/etc/kubernetes/audit-policy.yaml"
 nodes:
 # the control plane node
 - role: control-plane
+  extraMounts:
+  - containerPath: /etc/kubernetes/audit-policy.yaml
+  - hostPath: /tmp/audit-policy.yaml
 - role: worker
 - role: worker
 EOF
+    # create the audit-policy necessary for API Coverage
+    # https://kubernetes.io/docs/tasks/debug-application-cluster/audit/#audit-policy
+    cp $(dirname $0)/audit-policy.yaml > /tmp/audit-policy.yaml
```

Comment:
the redirection probably shouldn't be here, just CP?

Reply:
we should capture the dir earlier in the script as well

Reply:
The `>` was a typo, fixed.